addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -1205,6 +1207,16 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
      * @override
      */
     _renderCustomXML: async function (uiFragment) {
+        // In case the file visibility selector does not exist, create it.
+        if (!uiFragment.querySelector('we-select[data-name="hidden_condition_file_opt"]')){
```

Comment:
As said in my previous message, you can remove that `if` now that you removed the XML, right? It's always true :+1: